dotnet/src/webdriver/UnhandledAlertException.cs

```diff
@@ -27,11 +28,10 @@ namespace OpenQA.Selenium
     [Serializable]
     public class UnhandledAlertException : WebDriverException
     {
-        private string alertText;
-
         /// <summary>
         /// Initializes a new instance of the <see cref="UnhandledAlertException"/> class.
         /// </summary>
+        [Obsolete("Use a constructor overload that sets alertText")]
```

Comment:
It is unnecessary to deprecate some method. I guess we should understand whether `AlertText` might be null or not.

Reply:
Fixed